MantidQt/CustomInterfaces/src/Indirect/CalculatePaalmanPings.cpp

```diff
@@ -425,5 +421,30 @@ void CalculatePaalmanPings::addShapeSpecificCanOptions(IAlgorithm_sptr alg,
   }
 }
 
+/**
+ * Handles saving of workspace
+ */
+void CalculatePaalmanPings::saveClicked() {
+
+  checkADSForPlotSaveWorkspace(m_pythonExportWsName, false);
+  const auto outputWs = QString::fromStdString(m_pythonExportWsName);
+  addSaveWorkspaceToQueue(outputWs);
+  m_batchAlgoRunner->executeBatchAsync();
+}
```

Comment:
OK, I see here that the intended behaviour is what I thought. You just forgot to add the last line in `void ContainerSubtraction::saveClicked()`.